.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Contributor Code of Conduct
```

Comment:
Is this a copy from a different source? Is it from https://www.apache.org/foundation/policies/conduct ? Just curious in case we want to keep it up-to-date with the source material, if that gets updated.

Reply:
It is the same copy from: https://www.apache.org/foundation/policies/conduct. Do you say it by the title? Confusion of mine ...

Reply:
I just wanted to understand where it came from. You could mention it in the git commit log message. However, I think it'd be better to follow my proposal to create this in a central place for all ASF projects, instead of adding it to the Accumulo repository. See https://github.com/apache/accumulo/pull/2068#issuecomment-830654865